core/src/server/Server.cpp

```diff
@@ -253,14 +252,33 @@ Server::LoadConfig() {
     return milvus::Status::OK();
 }
 
-void
+Status
 Server::StartService() {
-    engine::KnowhereResource::Initialize();
+    Status stat;
+    stat = engine::KnowhereResource::Initialize();
+    if (!stat.ok()) {
+        std::cerr << "knowhere resource initialize fail: " << stat.message() << std::endl;
+        return stat;
+    }
+
     scheduler::StartSchedulerService();
-    DBWrapper::GetInstance().StartService();
+
+    stat = DBWrapper::GetInstance().StartService();
+    if (!stat.ok()) {
+        std::cerr << "DBWrapper start service fail: " << stat.message() << std::endl;
```

Comment:
change to:
SERVER_LOG_ERROR << "DBWrapper start service fail: " << stat.message();
std::cerr << "Milvus initializes fail: " << stat.message() << std::endl;